plutus-ledger/src/Ledger/CardanoWallet.hs

```diff
@@ -68,10 +69,10 @@ newtype WalletNumber = WalletNumber { getWallet :: Integer }
     deriving anyclass (FromJSON, ToJSON)
 
 fromWalletNumber :: WalletNumber -> MockWallet
-fromWalletNumber (WalletNumber i) = fromSeed (BSL.toStrict $ serialise i)
+fromWalletNumber (WalletNumber i) = fromSeed (BSL.toStrict $ serialise i) ""
```

Comment:
Use `noPassphrase` here.

Reply:
Adjusted, I used the ' version of the function that has no passphrase argument